### PR TITLE
Add Windows.EventLogs.Cleared

### PR DESCRIPTION
**Pull request by mgreen27**
name: Windows.EventLogs.Cleared

description: |
  Extract Event Logs related to EventLog clearing 
  - Security Log  - EventID 1102
  - System Log - EventID 104
  
reference:
  - https://attack.mitre.org/versions/v6/techniques/T1070/

image

### CODE REVIEW

File: artifacts/testdata/windows/evtx.out.yaml [hunk 1]
```diff
@@ -118,4 +118,29 @@ SELECT * FROM parse_evtx(filename=srcDir + '/artifacts/testdata/files/Security_1
...
+ {
+  "EventID": 104,
+  "Channel": "System",
+  "Message": "Log clear\r\n",
+  "Source": "\\\\.\\C:\\Windows\\System32\\winevt\\Logs\\System.evtx"
+ },
+ {
+  "EventID": 1102,
+  "Channel": "Security",
+  "Message": "The audit log was cleared.\nSubject:\n\tSecurity ID:\tS-1-5-21-2195900957-2730782454-3917332813-1001\n\tAccount Name:\tappveyor\n\tDomain Name:\tAPPVYR-WIN\n\tLogon ID:\t84282\r\n",
```
Comment (scudette) [inline artifacts/testdata/windows/evtx.out.yaml:143]: just wondering if the logon id is stable here? wont that number change?

File: artifacts/definitions/Windows/EventLogs/Cleared.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                                UserData.LogFileCleared.SubjectUserName,
+                            else= UserData.LogFileCleared.SubjectDomainName + '\\' +
+                                UserData.LogFileCleared.SubjectUserName) as UserName,
+                        if(condition= System.EventID.Value = 1102,        
+                            then= UserData.LogFileCleared.SubjectUserSid,
+                            else= System.Security.UserID) as SecurityID,    
+                        Message,
+                        if(condition=Source, then=Source, else=FullPath) as Source,
+                        format(format="%v-%v-%v",args=[System.EventID.Value,System.EventRecordID,
+                            timestamp(epoch=int(int=System.TimeCreated.SystemTime))]) as _Group
```
Comment (scudette) [inline artifacts/definitions/Windows/EventLogs/Cleared.yaml:81]: do you need the extra int here?

Comment (mgreen27) [reply artifacts/definitions/Windows/EventLogs/Cleared.yaml:81]: fixed and pushed.

Comment (mgreen27) [reply artifacts/definitions/Windows/EventLogs/Cleared.yaml]: @scudette I just thought, actually the int for the EventTime field is preferred due to it making the time a bit more friendly:
2020-06-08T05:58:09Z
vs
2020-06-08T05:58:09.212196826Z

**Pull request merged**